sentinelhub/api/batch.py

```diff
@@ -175,7 +176,11 @@ def get_tiling_grid(self, grid_id: Union[int, str]) -> Dict[str, Any]:
         :param grid_id: An ID of a requested tiling grid
         :return: A tiling grid definition
         """
-        return self.client.get_json(self._get_tiling_grids_url(grid_id), use_session=True)
+        url = self._get_tiling_grids_url(grid_id)
+        json_response = self.client.get_json(url=url, use_session=True)
+        if not isinstance(json_response, dict):
+            raise RuntimeError(f"Response from {url} did not contain the expected information.")
+        return json_response
```

Comment:
This part of the code now happens so many time that it could even become its own method on the common base class. E.g. `SentinelHubService.get_json_data()` or something like that.

Reply:
Will also be done in the next MR since this pattern also repeats itself in the API part.

Reply:
Actually I'll add it to this one, no reason to wait